test/functional/admin/promotional_feature_items_controller_test.rb

```diff
@@ -131,4 +131,14 @@ class Admin::PromotionalFeatureItemsControllerTest < ActionController::TestCase
     assert_not PromotionalFeatureItem.exists?(promotional_feature_item.id)
     assert_equal "Feature item deleted.", flash[:notice]
   end
+
+
+  test "GET :confirm_destroy calls correctly" do
```

Comment:
This commit looks good but it needs a rubocop please

Reply:
Ok I ran the rubocop..will check n run again no worries